test/config_bad_subjectaltnames.yaml

```diff
@@ -1,14 +1,20 @@
 ---
-logVLevel: 4
-subjectAltNames:
-  - localhost
-  - '10.43.0.1'
-  - '1.2.3.4'
-nodeName: node1
-nodeIP: '1.2.3.4'
 cluster:
-  url: https://1.2.3.4:6443
-  clusterCIDR: '10.20.30.40/16'
-  serviceCIDR: '40.30.20.10/16'
-  domain: cluster.local
+  name: 'my-cluster'
+  baseDomain: 'example.com'
+network:
+  clusterNetwork:
+    - cidr: '10.20.30.40/16'
+  serviceNetwork:
+    - '40.30.20.10/16'
   serviceNodePortRange: 30000-32767
+node:
+  hostnameOverride: node1
+  nodeIP: '1.2.3.4'
+apiServer:
+  subjectAltNames:
```

Comment:
I'm not gonna lie and say that the variable name as "subjectAltNames" indicates some sort of string name but the value is used as networking/IP info is just confusing. However, it is what it is so I make my peace with that.

Reply:
This is an example of "bad subject alt names" (by the file name `config_bad_subjectaltnames.yaml`) So I guess it's some test checking things that we should not use in that field

Reply:
One would expecdt subjectAltNames to be FQDNs, I agree. Seeing IPs here also only makes sense to me as this is for a unit test for a failure case.

Reply:
As SAN is clearly defined in the technical standard, using that name as param is the most descriptive IMO. It's just my ignorance that I didn't know the standard allowed IPs :)

I wouldn't do anything with the YAML. It's a unit test case, after all, not user documentation.